libs/common-types/src/lib/con-profile/con-profile.entityprops.ts

```diff
@@ -64,4 +64,5 @@ export class ConProfileEntityProps implements EntityProps {
   age?: number
   fullName: string
   doesNotHaveAvailableMentorshipSlot: boolean
+  availableMentorshipSlots: number
```

Comment:
@ericbolikowski, I don't understand why we need this new number field if we are not using its value in the code. Could you please clarify?

Reply:
Not needed. Solved 1:1.

Reply:
Nice, then we also need to remove it from Salesforce.

Reply:
It was already in Salesforce - I think it's a "FYI field", but maybe it's also used in some formulas